model/jpa/src/main/resources/META-INF/jpa-changelog-26.5.0.xml

```diff
@@ -88,4 +88,24 @@
         <dropColumn tableName="WORKFLOW_STATE" columnName="WORKFLOW_PROVIDER_ID"/>
     </changeSet>
 
+    <changeSet id="26.5.0-add-remember-me" author="keycloak">
+        <addColumn tableName="OFFLINE_USER_SESSION">
+            <column name="REMEMBER_ME" type="BOOLEAN" defaultValue="false">
+                <constraints nullable="false"/>
```

Comment:
To allow for a graceful migration of the user sessions that are remember me vs. non-remember me, I suggest to allow null values in the REMEMBER_ME column. When doing a cleanup of the rows that expired, one would select remember me, non-remember me, and null values. For the null values, it would trigger a conversion, and it could either write the updated remember me value to the database (on "true"), or delete the entry. 

Please let me know your thoughts.